package.json

```diff
@@ -7,22 +7,27 @@
   "main": "lib/cjs/index.js",
   "module": "lib/esm/index.js",
   "types": "lib/cjs/index.d.ts",
+  "packageManager": "pnpm@7.25.0",
+  "engines": {
+    "node": "18.x",
+    "pnpm": "7"
```

Comment:
```suggestion
```

This is redundant now that `packageManager` exists.

In fact its dangerous because we dont want it to be published to npm this way.